corehq/apps/data_dictionary/bulk.py

```diff
@@ -111,6 +113,11 @@ def _process_sheets(domain, workbook, allowed_value_info):
     data_type_map = {t.label: t.value for t in CaseProperty.DataType}
     seen_props = defaultdict(set)
     missing_valid_values = set()
+    data_validation_enabled = toggles.CASE_IMPORT_DATA_DICTIONARY_VALIDATION.enabled(domain)
+    if data_validation_enabled:
+        column_mapping = COLUMN_MAPPING + DATA_TYPE_MAPPING
```

Comment:
@zandre-eng It seems like the test fails here

```suggestion
        column_mapping = COLUMN_MAPPING | DATA_TYPE_MAPPING
```

Reply:
Ah, silly mistake! Thanks for pointing that out, it has been fixed now.